### PR TITLE
omap: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
DEVICE_TITLE is split up into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT

### CODE REVIEW

**Comment by mwarning** [comment]
@adrianschmutzler

File: target/linux/omap/image/Makefile [hunk 1]
```diff
@@ -44,29 +44,33 @@ endef
 #uboot-omap-am335x_evm uboot-omap-omap3_beagle uboot-omap-omap3_overo uboot-omap-omap4_panda
 
 define Device/ti_am335x-evm
-  DEVICE_TITLE := TI AM335x EVM
+  DEVICE_VENDOR := Texas Instruments
+  DEVICE_MODEL := TI AM335x EVM
```
Comment (adrianschmutzler) [inline target/linux/omap/image/Makefile]: Remove the TI here and below.

File: target/linux/omap/image/Makefile [hunk 2]
```diff
@@ -44,29 +44,33 @@ endef
...
   DEVICE_DTS := am335x-evm
 endef
 
 TARGET_DEVICES += ti_am335x-evm
 
 define Device/ti_am335x-bone-black
-  DEVICE_TITLE := TI AM335x BeagleBone Black
+  DEVICE_VENDOR := None
+  DEVICE_MODEL := TI AM335x BeagleBone Black
```
Comment (adrianschmutzler) [inline target/linux/omap/image/Makefile]: vendor: Texas Instruments
model: without TI

File: target/linux/omap/image/Makefile [hunk 3]
```diff
@@ -44,29 +44,33 @@ endef
...
   DEVICE_DTS := am335x-boneblack
 endef
 
 TARGET_DEVICES += ti_am335x-bone-black
 
 define Device/ti_omap4-panda
-  DEVICE_TITLE := OMAP4 TI pandaboard
+  DEVICE_VENDOR := None
+  DEVICE_MODEL := OMAP4 TI pandaboard
```
Comment (adrianschmutzler) [inline target/linux/omap/image/Makefile]: I'm not sure whether Texas instruments really is the vendor here or whether this device is just targetting the TI architecture ...

Comment (mwarning) [reply target/linux/omap/image/Makefile]: I think the vendor is PandaBoard.org.

Comment (mwarning) [reply target/linux/omap/image/Makefile]: I've added PandaBoard.org as vendor.

File: target/linux/omap/image/Makefile [hunk 4]
```diff
@@ -44,29 +44,33 @@ endef
...
   DEVICE_DTS := omap4-panda
   DEVICE_PACKAGES := kmod-usb-net-smsc95xx
 endef
 
 TARGET_DEVICES += ti_omap4-panda
 
 define Device/ti_omap3-beagle
-  DEVICE_TITLE := OMAP3 TI beagleboard
+  DEVICE_VENDOR := None
+  DEVICE_MODEL := OMAP3 TI beagleboard
```
Comment (adrianschmutzler) [inline target/linux/omap/image/Makefile]: What's the vendor here?

Comment (mwarning) [reply target/linux/omap/image/Makefile]: BeagleBoard.org

**Comment by mwarning** [comment]
@lynxis please review - thanks

**Comment by lynxis** [comment]
Thanks! @adrianschmutzler @mwarning

**Pull request closed without merging**